dev-docs/modules/dailyhuntBidAdapter.md

```diff
@@ -0,0 +1,104 @@
+# Overview
```

Comment:
You are missing the meta data. However I'm not sure what to put here as the adapter documentation is already available